starport-core/src/main/scala/com/krux/starport/db/record/ScheduledPipeline.scala

```diff
@@ -11,12 +11,14 @@ import java.time.LocalDateTime
 
 
 case class ScheduledPipeline(
-  awsId: String,
-  pipelineId: Int,
-  pipelineName: String,
-  scheduledStart: LocalDateTime,
-  actualStart: LocalDateTime,
-  deployedTime: LocalDateTime,
-  status: String,
-  inConsole: Boolean
-)
+                              awsId: String,
```

Comment:
please keep the original indentation (format is described here: https://github.com/salesforce/starport/blob/master/.scalafmt.conf)

Reply:
Project configured formatting applied.